verification/cbmc/stubs/s_remove_entry_override.c

```diff
@@ -0,0 +1,32 @@
+#include <aws/common/hash_table.h>
+#include <aws/common/math.h>
+#include <aws/common/private/hash_table_impl.h>
+#include <aws/common/string.h>
+
+#include <limits.h>
+#include <proof_helpers/make_common_data_structures.h>
+#include <proof_helpers/utils.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+/* Clears an entry. Does _not_ invoke destructor callbacks.
+ * Returns the last slot touched (note that if we wrap, we'll report an index
+ * lower than the original entry's index)
+ */
+size_t __CPROVER_file_local_hash_table_c_s_remove_entry(
```

Comment:
I'm tracking this stub with issue https://github.com/awslabs/aws-c-common/issues/709, so we can remove it after improving CBMC performance.

Reply:
Do we have a proof for this function that has the same pre/post conditions?

Reply:
We don't have a proof for this function. I created issue https://github.com/awslabs/aws-c-common/issues/711 to add this on another PR.

Reply:
OK